docs/docs/dev/security/certificates.md

```diff
@@ -28,102 +27,83 @@ Note, this CA is not the same CA for service-to-service communication.
 
 **To override autogenerated certificates**
 
-:::: tabs :options="{ useUrlFragment: false }"
-::: tab "Kubernetes (kumactl)"
 1) Prepare certificates
 
-Generate a TLS pair with a PKI of your choice and store it in PEM-encoded format in `/tmp/tls.crt`, `/tmp/tls.key`. Store the CA that was used to sign this pair in `/tmp/ca.crt`
+Generate a TLS pair with a PKI of your choice and store it in PEM-encoded format in `/tmp/tls.crt`, `/tmp/tls.key`. 
+Store the CA that was used to sign this pair in `/tmp/ca.crt`.
 
-We can also use `kumactl` to generate self-signed certs. 
+We can also use `kumactl` to generate self-signed certs:
 ```sh
 kumactl generate tls-certificate \
   --type=server \
-  --hostname=kuma-control-plane.kuma-system \ # adjust the name if you are installing Kuma to other namespace
-  --hostname=kuma-control-plane.kuma-system.svc \ # adjust the name if you are installing Kuma to other namespace
+  --hostname=<KUMA_CP_DNS_NAME> \
   --cert-file=/tmp/tls.crt \
   --key-file=/tmp/tls.key
-cp /tmp/tls.crt /tmp/ca.crt # since this is self-signed cert, the cert is also a CA
 ```
 
-2) Create a secret in the namespace where the control plane is installed
+Since "tls.crt" is a self-signed cert, it is also a CA:
+```sh
+cp /tmp/tls.crt /tmp/ca.crt
+```
+
+2) Configure the control plane with generated certs:
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes (kumactl)"
+Create a secret in the namespace where the control plane is installed:
 ```sh
-kubectl create secret generic general-tls-certs -n kuma-system \
+kubectl create secret generic general-tls-certs -n <namespace> \
   --from-file=tls.crt=/tmp/tls.crt \
   --from-file=tls.key=/tmp/tls.key \
   --from-file=ca.crt=/tmp/ca.crt
 ```
 
-3) Point to this secret when installing Kuma
+Point to this secret when installing Kuma:
```

Comment:
You get rid of the 3 but not the 1 and 2 any reason?

Reply:
I didn't try to get rid of numbers, it's just that 1 is now shared for all environments (kubectl, helm, and universal). Numbers 2 and 3 are environment-specific and that's why they're merged together. I made it look similar to the instructions below on that page.